pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -808,6 +846,11 @@ monitorLoop:
 				break monitorLoop
 			}
 
+			// Calculate the Migration Data Transfer Rate
```

Comment:
`transferRate` is just `progessDela/timeDela`,

transferRate=(progressWatermark-remainingData)/(now-lastProgressUpdate)
and you can add it to migrateInfoStats

Reply:
Hey Vladik, I do have a second commit for the derived metric here - https://github.com/shwetaap/kubevirt/commit/fc7c5e0540c995b94492000c962df9dd0ae9dcf6  
@fabian wanted to keep the two PRs different. So I will be creating another PR for it.

Reply:
> Hey Vladik, I do have a second commit for the derived metric here - [shwetaap@fc7c5e0](https://github.com/shwetaap/kubevirt/commit/fc7c5e0540c995b94492000c962df9dd0ae9dcf6)

:+1: seems like what we need.
@fabiand what's the reason for separating these?

Reply:
If you are fine in combining those, then I'm fine as well.